### DISCUSSION TITLE
1D formulation in 2D problems

### DISCUSSION THREAD

**Discussion by PetitPipi** [Anura3D/Anura3D_OpenSource#102] [Q&A] [answered]
Hi everyone,

I'd like to know that is there a way to form a 1D entity in a 2D problem in GiD and Anura3D (Or similarly 2D entities in 3D problems)? 
For example, a very thin layer of geogrid (a straight line in the model) inside the soils (2D continuum)?

Thanks!

**Comment by ayerrocolom** [comment, marked as answer]
Hello!
At this point, Anura3D cannot incorporate "bars" in 2D problems or "plates" or "membranes" in 3D problems. In such case, you will need to model such structures with a corresponding thickness and discretized with material points.
